hitl/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/bash -e
+
```

Comment:
`set -eux` is usually a good choice. Otherwise commands will silently fail and not be printed.